packages/xo-web/src/common/intl/messages.js

```diff
@@ -1055,13 +1055,16 @@ const messages = {
   hostCpusNumber: 'Core (socket)',
   hostManufacturerinfo: 'Manufacturer info',
   hostBiosinfo: 'BIOS info',
+  hostUsbs: 'USBs',
   licenseHostSettingsLabel: 'License',
   hostLicenseType: 'Type',
   hostLicenseSocket: 'Socket',
   hostLicenseExpiry: 'Expiry',
   hostRemoteSyslog: 'Remote syslog',
   hostIommu: 'IOMMU',
   hostNoCertificateInstalled: 'No certificates installed on this host',
+  passthroughEnabled: 'Enable passthrough',
```

Comment:
`passthrough Enabled` seems to suggest that the `passthrough is enabled`. On the other hand, `Enable passthrough` seems more like an action. It means `you are going to enable the passthrough’ (we assume the passthrough is not enabled here)

Reply:
I wanted to use this message for the toggle buttons column